webpack/assets/javascripts/react_app/ReactApp/ReactAppHelper.js

```diff
@@ -0,0 +1,5 @@
+import store from '../redux';
+import { APP_FETCH_SERVER_PROPS } from '../constants';
+
+export const initializeMetadata = metadata =>
```

Comment:
Can't it be an action? It feels like a workaround redux when it can simply be implemented with redux best practices.